### PR TITLE
Parse stream info consistently

### LINKED ISSUE #64: Opening .xdf generated by LabRecorder
Hello.
My name is Nelson.
I managed to run LabRecorder on my RASPBERRY PI 4.
However, opening the generated .xdf with mnelab is not possible.
Find attached error and file.
Thanks a lot.
Capture
[foo.zip](https://github.com/xdf-modules/xdf-python/files/5038060/foo.zip)

### PR DESCRIPTION
**Pull request by agricolab**
Parsing and decoding was handled differently in load_xdf
versus parse_chunks. This commit aligns the behavior to the
load_xdf implementation, because sometimes, the casting as int
fails because the str is not an int literal

Should also resolve #64

### CODE REVIEW

**Comment by agricolab** [comment]
Probably better to postpone until the pep8 PR https://github.com/xdf-modules/xdf-python/pull/63 was merged.

File: pyxdf/pyxdf.py [hunk 1]
```diff
@@ -736,6 +736,7 @@ def parse_chunks(chunks):
     streams = []
     for chunk in chunks:
         if chunk["tag"] == 2:  # stream header chunk
+        	# if you edit, check for consistency with parsing in load_xdf
```
Comment (cbrnr) [inline pyxdf/pyxdf.py:739]: This is not correctly indented.

File: pyxdf/pyxdf.py [hunk 2]
```diff
@@ -774,7 +775,9 @@ def _read_chunks(f):
         if chunk["tag"] in [2, 3, 4, 6]:
             chunk["stream_id"] = struct.unpack("<I", f.read(4))[0]
             if chunk["tag"] == 2:  # parse StreamHeader chunk
-                xml = ET.fromstring(f.read(chunk["nbytes"] - 6).decode())
+            	# if you edit, check for consistency with parsing in load_xdf
```
Comment (cbrnr) [inline pyxdf/pyxdf.py:778]: Same as before (not correctly indented).

File: pyxdf/pyxdf.py [hunk 3]
```diff
@@ -774,7 +775,9 @@ def _read_chunks(f):
...
+                msg = f.read(chunk["nbytes"] - 6).decode('utf-8', 'replace')
```
Comment (cbrnr) [inline pyxdf/pyxdf.py:779]: Black wants double quotes for strings.

**Comment by agricolab** [comment]
I implemented the changes again on the tip of `master`, which now includs pep8 formatting @cbrnr implemented. The reviewed wrong indentation was corrected, and i run it through black with -l 79. I can only assume the wrong formatting earlier came from me not using my IDE, as the IDE autoformats, so i am happy the color is now black. I decided to force-push, as the changes from `pep8` would have been too numerous to warrant working through the merge-conflict with  `roundfloat`.

**Comment by cbrnr** [comment]
Final request: can you add a changelog entry please?

**Comment by cbrnr** [comment]
I added the changelog entry. Thanks @agricolab!

**Pull request merged**